pkg/apphealth/health.go

```diff
@@ -197,6 +197,7 @@ func (h *AppHealth) doProbe(parentCtx context.Context) {
 	successful, err := h.probeFn(ctx)
 	// In case of errors, we do not record the failed probe because this is generally an internal error
```

Comment:
This comment needs to be updated (or removed).

Reply:
Removed